docs/src/taylorize.md

```diff
@@ -1,18 +1,18 @@
 # [Optimizing: `@taylorize`](@id taylorize)
 
 Here, we describe the use of the macro [`@taylorize`](@ref), which
-parses the functions containing the ODEs to be integrated, allowing to speed up
-[`taylorinteg`](@ref) and [`lyap_taylorinteg`](@ref).
+parses the functions containing the ODEs to be integrated, allowing
+[`taylorinteg`](@ref) and [`lyap_taylorinteg`](@ref) to be sped up.
```

Comment:
I think this should read "to be speed up".

Reply:
My pleasure, this is the least I can do.

On this line, it could be either "sped" or "speeded" as in the intransitive form of speed.  While not authoritative, you can see from https://english.stackexchange.com/questions/114620/speeded-vs-sped how uncommon it is to use "speeded".  Anecdotally, my web browser is highlighting a supposed spelling error in "speeded", indicating that it's so uncommon that it's not in the dictionary the browser is using.

Reply:
I stand corrected... I checked against a dictionary, and [sped](https://www.merriam-webster.com/dictionary/SpEd) is ok :-)

I'll merge this.